src/lib.rs

```diff
@@ -137,3 +137,20 @@ impl<BE> From<PdError> for Error<BE> {
         Error::Pd(err)
     }
 }
+
+/// Zero-sized type for use by [`TryFrom`] and similar where [`PdError`] is too broad
```

Comment:
If I got an error of this type from a function, I'd have no idea from this description what it really means.

```suggestion
/// The input was invalid and could not be converted the expected type.
///
/// See function documentation for specifics on what this error represents.
```

Reply:
Done.